docs/tutorials/sqlalchemy/0-introduction.rst

```diff
@@ -96,8 +96,8 @@ serializable by Litestar.
 Behavior
 ++++++++
 
-the ``add_item()`` and ``update_item()`` routes no longer return the full collection, instead they return the item that
-was added or updated. This is a minor detail change, but it is worth noting as it brings the behavior of the app closer
+The ``add_item()`` and ``update_item()`` routes no longer return the full collection, instead they return the item that
+was added or updated. This is a minor detail change, but it is worth nothing as it brings the behavior of the app closer
```

Comment:
I think we actually mean `noting` here and not `nothing`.  Maybe it should be `it is worth mentioning`

Reply:
Thankyou @cofin. Review latest changes.